wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/convert/component/plugin/PluginComponentToWorkspaceApplier.java

```diff
@@ -99,15 +97,6 @@ public void apply(
           .put(format(SIDECAR_MEMORY_LIMIT_ATTR_TEMPLATE, fqn.getPublisherAndName()), memoryLimit);
     }
 
-    for (Entry<String, String> preference : pluginComponent.getPreferences().entrySet()) {
-      workspaceConfig
-          .getAttributes()
-          .put(
-              format(
-                  PLUGIN_PREFERENCE_ATTR_TEMPLATE, fqn.getPublisherAndName(), preference.getKey()),
```

Comment:
Please consider removing `Constants.PLUGIN_PREFERENCE_ATTR_TEMPLATE` at all since seems it's unused now.

Reply:
ok